### PR TITLE
Add 8 new models and updates

### PR DESCRIPTION
**Pull request by pedropaulofb**
Eight new models added. All models were sent via the catalog's Google Forms after we got in contact with the authors.
- alkhalaf2024
- barcelos2024resiliont
- elghosh2024eucaim
- nascimento2023spacecon
- piest2024dsr-kb
- pinheiro2024api
- sariev2024maritime
- schoonderbeek2024eamon
 
Model updated:
- xhani2023xmlpo

All `ttl` files with the outdated `purl.org` Catalog URIs have been updated to the latest `w3id.org` URI.

### CODE REVIEW

File: models/alkhalaf2024/metadata.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+ - domain
+language: en
+designedForTask:
+ - conceptual clarification
+ - interoperability
+ - ontological analysis
+context:
+ - research
+source:
+ - https://tinyurl.com/5n8c7k8t
```
Comment (tgoprince) [inline models/alkhalaf2024/metadata.yaml:25]: We should not use a GoogleDrive link with some images as the source. It should, instead, be the paper which includes these figures. 

http://doi.org/10.5220/0012183900003598

File: models/alkhalaf2024/metadata.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+representationStyle:
+ - ontouml
+landingPage:
+ - https://hdl.handle.net/11311/1256092
```
Comment (tgoprince) [inline models/alkhalaf2024/metadata.yaml:29]: The landing page field should not point to a paper. The source field should be used for that.

File: models/barcelos2024resiliont/metadata.yaml [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+ontologyType:
+ - core
+language: en
+designedForTask:
+ - conceptual clarification
+ - ontological analysis
+context:
+ - research
+source:
+ - https://raw.githubusercontent.com/pedropaulofb/resiliont/main/resources/Ontological%20Foundations%20of%20Resilience.pdf
```
Comment (tgoprince) [inline models/barcelos2024resiliont/metadata.yaml:28]: Replace this link with the DOI of the paper.

File: models/elghosh2024eucaim/metadata.yaml [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+context:
+ - research
+ - industry
+source:
+ - https://www.utwente.nl/en/eemcs/fois2024/resources/papers/el-ghosh-et-al-towards-semantic-interoperability-among-heterogeneous-cancer-data-models-using-a-layered-modular-hyper-ontology.pdf
+ - https://www.utwente.nl/en/eemcs/fois2024/resources/papers/el-ghosh-et-al-grounding-a-hyper-ontology-on-mcode-ontological-conceptual-model-and-foundational-ontologies-for-semantic-interoperability-in-the-oncology-domain.pdf
+representationStyle:
+ - ontouml
+landingPage:
+ - https://catalogue.eucaim.cancerimage.eu/#/
```
Comment (tgoprince) [inline models/elghosh2024eucaim/metadata.yaml:36]: This does not seem to be the landing page of the ontology.

File: models/pinheiro2024api/metadata.yaml [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+ - https://dblp.org/pid/306/9640
+ - https://dblp.org/pid/38/10199
+ - https://dblp.org/pid/217/4594
+keyword:
+ - api
+ - automatic architecture modelling
+ - enterprise architecture
+ - enterprise architecture management
+ - mining
+ - ontology
```
Comment (tgoprince) [inline models/pinheiro2024api/metadata.yaml:15]: Ontology as a keyword is pointless.

File: models/pinheiro2024api/metadata.yaml [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+editorialNote:
+ontologyType:
+ - domain
+language: en
+designedForTask:
+ - conceptual clarification
+context:
+ - research
+source:
+ - https://link.springer.com/chapter/10.1007/978-3-031-64755-0_13
```
Comment (tgoprince) [inline models/pinheiro2024api/metadata.yaml:26]: Use the paper's DOI instead.

File: models/sariev2024maritime/metadata.yaml [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+ - interoperability
+ - decision support system
+context:
+ - research
+source:
+ - http://essay.utwente.nl/101204/1/Sariev_BA_EEMCS.pdf
+representationStyle:
+ - ontouml
+landingPage:
+ - https://essay.utwente.nl/101204/
```
Comment (tgoprince) [inline models/sariev2024maritime/metadata.yaml:28]: This is not a landing page. Please remove.

File: models/sariev2024maritime/metadata.yaml [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+ - domain
+language: en
+designedForTask:
+ - conceptual clarification
+ - interoperability
+ - decision support system
+context:
+ - research
+source:
+ - http://essay.utwente.nl/101204/1/Sariev_BA_EEMCS.pdf
```
Comment (tgoprince) [inline models/sariev2024maritime/metadata.yaml:24]: You should use the permanent url provided by the UT: https://purl.utwente.nl/essays/101204

File: models/schoonderbeek2024eamon/metadata.yaml [hunk 9]
```diff
@@ -0,0 +1,29 @@
...
+ - conceptual clarification
+ - ontological analysis
+context:
+ - research
+source:
+ - https://doi.org/10.1007/s10270-023-01146-w
+representationStyle:
+ - ontouml
+landingPage:
+ - https://link.springer.com/article/10.1007/s10270-023-01146-w
```
Comment (tgoprince) [inline models/schoonderbeek2024eamon/metadata.yaml:27]: This is not a landing page for the ontology.

**Review by tgoprince — CHANGES_REQUESTED** [review]
Please see my comments on the files.

File: models/alkhalaf2024/metadata.yaml (hunk 2 above)
Comment (pedropaulofb) [reply models/alkhalaf2024/metadata.yaml:29]: Done.
source substituted to https://dx.doi.org/10.5220/0000178400003598
landingPage removed.

File: models/barcelos2024resiliont/metadata.yaml (hunk 3 above)
Comment (pedropaulofb) [reply models/barcelos2024resiliont/metadata.yaml:28]: There is no DOI yet. We can update the field once we have this piece of data.

File: models/elghosh2024eucaim/metadata.yaml (hunk 4 above)
Comment (pedropaulofb) [reply models/elghosh2024eucaim/metadata.yaml:36]: URL removed.

File: models/nascimento2023spacecon/metadata.yaml [hunk 10]
```diff
@@ -0,0 +1,29 @@
...
+ontologyType:
+ - core
+language: en
+designedForTask:
+ - conceptual clarification
+ - interoperability
+context:
+ - research
+source:
+ - https://link.springer.com/chapter/10.1007/978-3-031-47262-6_19
```
[the remark this answers is not in the document]
Comment (pedropaulofb) [reply models/nascimento2023spacecon/metadata.yaml:23]: Done.

File: models/pinheiro2024api/metadata.yaml (hunk 5 above)
Comment (pedropaulofb) [reply models/pinheiro2024api/metadata.yaml:15]: keyword removed.

**Comment by pedropaulofb** [comment]
@tgoprince, thank you for the review. All recommended adjustments have been made. Can you please review the PR again?

The modifications made were basically:
- fix all purl.org URIs (except from DC terms)
- fix reported landingPages and source pages
- minor fixes resultant from the transformation to ttl

**Comment by pedropaulofb** [comment]
@tgoprince , I fixed the mgic2011 dataset, which had non-OntoUML models. You can review it now.

File: models/alkhalaf2024/metadata.yaml (hunk 1 above)
Comment (tgoprince) [reply models/alkhalaf2024/metadata.yaml]: O DOI que você adicionou não resolve para o artigo, mas para o proceedings que inclui o artigo.

**Comment by tgoprince** [comment]
I found two URIs for MIT licenses: 
- https://mit-license.org/
- https://opensource.org/license/mit

Let's stick with one of them?

**Comment by tgoprince** [comment]
The file `sariev2024/metadata.yaml` contains a google drive link as a landing page: 
https://drive.google.com/drive/folders/1Y0ZbJ1RgKvVJZWORznzWLptFPQN5G2m5

The link also resolves to a folder containing only some diagrams in png files.

This does not fit the definition of landing page that we agreed in this project.

**Comment by pedropaulofb** [comment]
> I found two URIs for MIT licenses:
> 
> * https://mit-license.org/
> * https://opensource.org/license/mit
> 
> Let's stick with one of them?

Yes. Thanks for noticing. The correct one is https://opensource.org/license/mit. 

I am making the necessary substitutions.

**Comment by pedropaulofb** [comment]
> The file `sariev2024/metadata.yaml` contains a google drive link as a landing page: https://drive.google.com/drive/folders/1Y0ZbJ1RgKvVJZWORznzWLptFPQN5G2m5
> 
> The link also resolves to a folder containing only some diagrams in png files.
> 
> This does not fit the definition of landing page we agreed upon for this project.

The DCAT 3's [definition of landing page](https://www.w3.org/TR/vocab-dcat-3/#Property:resource_landing_page) is:

> A Web page that can be navigated to in a Web browser to gain access to the catalog, a dataset, its distributions and/or additional information.

The Google Drive's page do contain additional information about the model. Shouldn't we consider it as a valid landing page? Please reconsider.

File: models/alkhalaf2024/metadata.yaml (hunk 1 above)
Comment (pedropaulofb) [reply models/alkhalaf2024/metadata.yaml]: Thanks for noticing! Fixed.

**Comment by pedropaulofb** [comment]
@tgoprince, I addressed all the mentioned problems. Please check whether I should remove of not [sariev2024's landing page](https://github.com/OntoUML/ontouml-models/pull/286#issuecomment-2417026777). If you still believe it is the best option, let me know and then I will remove it asap.

**Comment by pedropaulofb** [comment]
> @pedropaulofb, the Google Drive link redirects to a page contain tree pdfs, each of which contain a single diagram. It does not even provide access to the actual file containing the model. Moreover, Google Drive links are very easily broken. I honestly, do not see any value of keeping this link as is.
> 
> I suggest we reject temporary URLs like this one all the metadata fields we collect.

Ok! I already removed the URL from the metadata.

**Review by tgoprince — APPROVED** [review]

**Pull request merged**